.github/workflows/nightly-tests.yml

```diff
@@ -54,6 +66,13 @@ jobs:
           vault-token: ${{ steps.vault-token-step.outputs.vault-token }}
           target: wsmtest
 
+      - name: ArgoCD sync
+        run: |
+          curl --fail --silent --show-error --location --request POST \
```

Comment:
Is the ArgoCD sync synchronous? I'm wondering how to tell if the sync was successful. Perhaps it could be helpful to call the WSM version endpoint before/after this so we can look at logs to see if an expected sync occurred.

Reply:
Added a sleep and version logging on either side of the argo trigger.